source/assets/stylesheets/examples/article-layout.scss

```diff
@@ -0,0 +1,38 @@
+@import "neat";
+
+$article-layout-grid: (
+  columns: 1,
+  gutter: 2rem,
+);
+
+$article-layout-grid--tablet-up: (
+  columns: 5,
+  gutter: 2rem,
+  media: "(min-width: 768px)",
+);
+
+.article-layout {
+  @include grid-container($article-layout-grid);
```

Comment:
@whmii it looks like it does, but is not used. I'll remove from docs and PR on neat